src/framework/shortcuts/view/shortcutsmodel.cpp

```diff
@@ -150,6 +150,7 @@ QVariant ShortcutsModel::currentShortcut() const
 
     QVariantMap obj;
     obj["title"] = actionTitle(sc.action);
+    obj["icon"] = static_cast<int>(action(sc.action).iconCode);
```

Comment:
I think it would be good to rewrite this as:
```suggestion
    const UiAction& action = this->action(sc.action);

    QVariantMap obj;
    obj["title"] = action.title;
    obj["icon"] = static_cast<int>(action.iconCode);
```
That way, you look up the `UiAction` only once, instead of twice (first through `actionTitle()`, then directly).